src/SFA.DAS.ApprenticeFeedback.Data.UnitTests/ApprenticeFeedbackRepository/WhenAddingApprenticeFeedbackTarget.cs

```diff
@@ -16,14 +16,14 @@ namespace SFA.DAS.ApprenticeFeedback.Data.UnitTests
     public class WhenAddingApprenticeFeedbackTarget
     {
         private Mock<IApprenticeFeedbackDataContext> _dbContext;
-        private ApprenticeFeedbackRepository _repository;
+        private Repository.ApprenticeFeedbackRepository _repository;
```

Comment:
This prefixing of Repository isn't needed ( usually ) can you see why it's required? and what you can change to make it so that it isn't required and should just be
`private ApprenticeFeedbackRepository _repository`

Reply:
Is it because it's in a different project? If so, then a using statement should fix it.

Reply:
Or is it because it inherits from the interface?

Reply:
It's down to the namespace mimicking the name of the type - ApprenticeFeedbackRepository. So it gets confused needs to be told specifically the type with the namespace prefix